### DISCUSSION TITLE
Handle compiler with different absolute paths

### DISCUSSION THREAD

**Discussion by tru** [ccache/ccache#250] [Support]
I am currently investigating why we get 100% cache misses in our setup. My first thought was that it was connected to the COMPILERCHECK since it by default uses the `mtime` attribute. So I switched to `content` but I still get the same 100% cache miss.

I currently suspect that we are getting all misses because in our CI setup we deploy the compiler we want to use to the current workspace at startup. That makes it easy for us to switch / upgrade compilers - but it also makes the compiler path change for every build. By my cursory look in the documentation and the source I didn't see anything that could solve my problem here. Is there a configuration option that would allow me to skip including the full path to the compiler in the hash?

**Comment by afbjorklund** [comment]
The default setup will only include the mtime (like you say) in the hash, not the path.

We probably need #226

**Comment by tru** [comment]
You are probably right - I will try the undocumented method to see if I can figure out why we don't get any hits at all.
